hr_employee_seniority/models/hr_employee.py

```diff
@@ -0,0 +1,85 @@
+# copyright 2013 Michael Telahun Makonnen <mmakonnen@gmail.com>
+# copyright 2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import calendar
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class HrEmployee(models.Model):
+
+    _inherit = 'hr.employee'
+
+    initial_employment_date = fields.Date(
+        string='Initial Date of Employment',
+        help='Date of first employment if it was before the start of the '
+             'first contract in the system.',
+    )
+    length_of_service = fields.Float(
+        'Months of Service',
+        compute='_compute_months_service',
+    )
+
+    def _first_contract(self):
+        Contract = self.env['hr.contract'].sudo()
```

Comment:
Could you fix the variable name, please?

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#variable-names